src/components/CharacterComponent.jsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import styled from "styled-components";
+
+import CharacterImg from "../shared/assets/fotoKim.jpeg";
+
+let scoreBar = "";
+let percentage = scoreBar;
+
+function getProgressInColumns(percentage) {
+  return 2 + percentage / 10;
```

Comment:
waar komt magic number 2 vandaan? als dat nodig is misschien even uitleggen met een comment of een constante binnen de functie van maken.

Reply:
magic 2 is nodig om het grid iets verder op te schuiven zodat het klopt met de range, ik zal een comment toevoeggen